File: app/src/main/java/org/ionc/wallet/callback/OnUnbindDeviceButtonClickedListener.java
package org.ionc.wallet.callback;


/**
 * USER: binny
 * DATE: 2018/9/14
 * 描述: 长按解绑设备
 */
public interface OnUnbindDeviceButtonClickedListener {
    void onUnbindButtonClick(String cksn, int position);
}

File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnUnbindDeviceButtonClickedListener.kt
package org.ionchain.wallet.mvp.callback

/**
 * USER: binny
 * DATE: 2018/9/14
 * 描述: 长按解绑设备
 */
interface OnUnbindDeviceButtonClickedListener {
    fun onUnbindButtonClick(cksn: String, position: Int)
}
